charts/minikube-values.yaml

```diff
@@ -15,23 +15,29 @@ ingress:
 global:
   gitlab:
     clientSecret: e3ecd740-0892-4610-ad69-c7f4886f4533
-    postgresPassword: 3b0aa4bae4fbbe167143f583e35eda18
-    sudoToken: 214981ee49479ecbee56359b70ad25dc87edd0e9ea569b7b8c522c3bf979446a
+    postgresPassword:
```

Comment:
Wait, why are the hardcoded values back here? Didn't you remove those @Panaetius?

Reply:
I'd removed them from the files in `example-configurations`, not this one (must have missed it). Though maybe this should be in `example-configurations` as well?

Reply:
The hardcoded values have been removed from all example configurations by Pamela's recent PR anyway. The example configurations for deployments against existing GitLab are now much more minimal and just contain the which _differ_ from other deployments.

Reply:
>Though maybe this should be in example-configurations as well?
not needed. Those files are good as they are and they cover the minimal set of values to overwrite minikube-values.yaml values.
I was actually going to comment on that, we should remove as many hardcoded passwords as we can from the minikube configuration.